### PR TITLE
golden-cheetah: Add libusb (ANT+ sensors), 3.5-DEV1903 -> 3.5-RC2X

### PR DESCRIPTION
**Pull request by JohnAZoidberg**
###### Motivation for this change
I wanted to use ANT+ sensors with GoldenCheetah, now with libusb it's possible.
I tested it using the ANT USB-Stick-m and a handful of ANT+ sensors - works flawlessly.

While at it, I upgraded the version.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

###### Notify maintainers

cc @ocharles

### CODE REVIEW

**Review by ocharles — APPROVED** [review]
Awesome, thanks!

File: pkgs/applications/misc/golden-cheetah/default.nix [hunk 1]
```diff
@@ -56,8 +56,7 @@ in mkDerivation rec {
     runHook postInstall
   '';
 
-  # RCC: Error in 'Resources/application.qrc': Cannot find file 'translations/gc_fr.qm'
-  enableParallelBuilding = false;
+  enableParallelBuilding = true;
```
Comment (veprbl) [inline pkgs/applications/misc/golden-cheetah/default.nix]: Can remove this line completely, already set in setupHook:
https://github.com/NixOS/nixpkgs/blob/323c2ba6b7c64ce942ec33b1e9545c352baa4404/pkgs/development/libraries/qt-5/hooks/qmake-hook.sh#L26-L29

**Review by veprbl — COMMENTED** [review]
Also, could we add
```
substituteInPlace src/gcconfig.pri \
  --replace "#DEFINES += NOWEBKIT" "DEFINES += NOWEBKIT"
```
And replace `qtwebkit` with `qtwebengine`?

**Comment by JohnAZoidberg** [comment]
Good idea, thanks!

**Pull request merged**